testsuite/tests/lib-unix/common/gethostbyname.ml

```diff
@@ -0,0 +1,14 @@
+(* TEST
+* hasunix
+include unix
+** bytecode
+** native
+*)
+
+(* Checks that gethostbyname supports both IPv4 and IPv6 (see #11461) *)
```

Comment:
gethostbyname -> gethostbyaddr.  Same thing in the filename for this test...

Reply:
Ah, my bad. Fixed in the new version.